wger/exercises/tests/test_exercise.py

```diff
@@ -391,6 +392,19 @@ def test_search_exercise_logged_in(self):
         self.user_login('test')
         self.search_exercise()
 
+    def test_show_all_excercise_infos(self):
+        response = self.client.get('/api/v2/exercise/special/')
+        self.assertIsInstance(response.data, list)
+        self.assertTrue(len(response.data), 10)
```

Comment:
How about doing something like this instead of hardcoding `10`, `self.assertTrue(len(response.data), len(Exercise.objects.all()))` It's more dynamic.